tools/istio-iptables/pkg/cmd/root.go

```diff
@@ -80,22 +80,24 @@ var rootCmd = &cobra.Command{
 			}
 		}
 		if cfg.RunValidation {
-			hostIP, err := getLocalIP()
+			hostIPs, err := getLocalIPs()
 			if err != nil {
 				// Assume it is not handled by istio-cni and won't reuse the ValidationErrorCode
 				panic(err)
 			}
-			validator := validation.NewValidator(cfg, hostIP)
+			for _, hostIP := range hostIPs {
+				validator := validation.NewValidator(cfg, hostIP)
```

Comment:
should this be split out? It seems orthogonal?

Reply:
And explain why we need to run it multiple times. I doubt we do. We just need to check that *a single IPtables rule was applied*; we aren't validating every rule.

Reply:
Hi @howardjohn, okay, got it, I remember I have changed it before, and just check one time

Reply:
Hi @howardjohn , change to validate the first hostIP.